integration-tests/runtime-tests/test/tests/oracle/HAL01-Tests.ts

```diff
@@ -224,17 +224,17 @@ describe("HAL01 [Oracle] Tests", function () {
       const maliciousPrice = api.createType("u128", 900);
       const asset = api.createType("u128", assetID);
 
-      const balanceWallet1BeforeTransaction = new BN(
-        (await api.rpc.assets.balanceOf(asset.toString(), walletHAL01_1.publicKey)).toString()
+      const oracleStakeWallet1BeforeTransaction = new BN(
+        (await api.query.oracle.oracleStake(walletHAL01_1.publicKey)).toString()
```

Comment:
looks like, a helper-function will useful
```
const oracleStake = async (wallet: KeyringPair) => {
  return new BN(
    (await api.query.oracle.oracleStake(wallet.publicKey)).toString()
   );
}
```
then we can simplify
```
const oracleStakeWallet1BeforeTransaction = await oracleStake(walletHAL01_1);
const oracleStakeWallet2BeforeTransaction = await oracleStake(walletHAL01_2); 
// ...

const oracleStakeWallet1AfterTransaction = await oracleStake(walletHAL01_1);
const oracleStakeWallet2AfterTransaction = await oracleStake(walletHAL01_2);
// ...

```

Reply:
Makes sense, will update this. 👍 
Thanks a lot. 🙏